### PR TITLE
Fix broken links - Feb 2022

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to dotnet/docs#27892.

### CODE REVIEW

**Comment by gewarren** [comment]
@adegeo Can you review this one?

File: xml/System.Windows.Controls/UserControl.xml [hunk 1]
```diff
@@ -47,7 +47,7 @@
   
  ]]></format>
     </remarks>
-    <related type="ExternalDocumentation" href="https://go.microsoft.com/fwlink/?LinkID=160025">NumericUpDown Custom Control with Theme and UI Automation Support Sample</related>
+    <related type="ExternalDocumentation" href="https://github.com/dotnet/docs-desktop/tree/main/dotnet-desktop-guide/samples/snippets/csharp/VS_Snippets_Wpf/UserControlNumericUpDown/CSharp">NumericUpDown Custom Control with Theme and UI Automation Support Sample</related>
```
Comment (adegeo) [inline xml/System.Windows.Controls/UserControl.xml:50]: This one should be deleted. We shouldn't deep link into the snippets area, only samples.

File: xml/System.Windows.Data/RelativeSource.xml [hunk 2]
```diff
@@ -45,7 +45,7 @@
   
  :::code language="xaml" source="~/snippets/csharp/System.Windows/FrameworkElement/DefaultStyleKey/themes/generic.xaml" id="Snippetrelativesource":::  
   
- For the complete sample, see [NumericUpDown Custom Control with Theme and UI Automation Support Sample](https://go.microsoft.com/fwlink/?LinkID=160025).  
+ For the complete sample, see [NumericUpDown Custom Control with Theme and UI Automation Support Sample](https://github.com/dotnet/docs-desktop/tree/main/dotnet-desktop-guide/samples/snippets/csharp/VS_Snippets_Wpf/UserControlNumericUpDown/CSharp).
```
Comment (adegeo) [inline xml/System.Windows.Data/RelativeSource.xml:48]: This one should be deleted. We shouldn't cross link into other repo snippet files, only samples.

File: xml/System.Windows.Data/RelativeSource.xml [hunk 3]
```diff
@@ -594,7 +594,7 @@
...
```
Comment (adegeo) [inline xml/System.Windows.Data/RelativeSource.xml:597]: This one should be deleted. We shouldn't cross link into other repo snippet files, only samples.

File: xml/System.Windows/FrameworkElement.xml [hunk 4]
```diff
@@ -1353,17 +1353,17 @@
...
+  The complete source code for this example is available on GitHub for both [C#](https://github.com/dotnet/docs-desktop/tree/main/dotnet-desktop-guide/samples/snippets/csharp/VS_Snippets_Wpf/UserControlNumericUpDown/CSharp) and [Visual Basic](https://github.com/dotnet/dotnet-api-docs/tree/main/snippets/visualbasic/VS_Snippets_Wpf/CustomControlNumericUpDown/visualbasic/customcontrollibrary).
 
  ]]></format>
         </remarks>
         <altmember cref="P:System.Windows.FrameworkElement.OverridesDefaultStyle" />
-        <related type="ExternalDocumentation" href="https://go.microsoft.com/fwlink/?LinkID=160025">NumericUpDown Custom Control with Theme and UI Automation Support Sample</related>
+        <related type="ExternalDocumentation" href="https://github.com/dotnet/docs-desktop/tree/main/dotnet-desktop-guide/samples/snippets/csharp/VS_Snippets_Wpf/UserControlNumericUpDown/CSharp">NumericUpDown Custom Control with Theme and UI Automation Support Sample</related>
```
Comment (adegeo) [inline xml/System.Windows/FrameworkElement.xml:1366]: These links should be altered too, perhaps to the local snippet, but not cross-repo.

**Review by adegeo — APPROVED** [review]
Some minor things for you to fix before merging.

**Pull request merged**